[18.0][MIG] mrp_unbuild_subcontracting: Migration to 18.0

xaviedoanhduy opened the pull request.

pedrobaeza committed:
[IMP] Total refactorization of quality control modules with new API, README files, and new concepts. WIP of quality_control_tolerance

Merge branch '8.0' of github.com:odoomrp/odoomrp-wip into 8.0

[FIX] quality_control_stock:
* File missing
* Triggers for product category

oihane committed:
[FIX] Little changes in quality modules

pedrobaeza committed:
[FIX] quality_control_stock:
* Omit lot when it's not present
* Change icon set

oihane committed:
[MOD] Trigger lines instead of test

pedrobaeza committed:
[IMP] Expand AUTHORS

oihane committed:
[MOD] Smartbuttons all in one div

pedrobaeza committed:
[IMP] quality_control_*: Make use of qty field

Making it a regular field, you can put any manual quantity in inspections

oihane committed:
[IMP] quality_control_stock: if lot in reference, get its info
quality_control_mrp: tests extended

[IMP] quality_control_mrp: removed not required code

agaldona committed:
[IMP] quality_control_stock, quality_control_mrp: set correct qty to tests

JordiBForgeFlow committed:
[9.0][ADD] mrp_production_putaway_strategy (#215)

* [9.0][ADD] mrp_production_putaway_strategy

* small fix

* fixes

* [FIX] travis and readme.

* [FIX] adapt tests to new logic

oca-transbot committed:
OCA Transbot updated translations from Transifex

LoisRForgeFlow committed:
[10.0][MIG] mrp_production_putaway_strategy

LoisRForgeFlow committed:
[11.0][MIG] mrp_production_putaway_strategy

SimoRubi committed:
[MIG] quality_control_stock: Migration to 10.0

NachoAlesLopez committed:
[MIG] quality_control_stock: Migration to 11.0

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update mrp_production_putaway_strategy.pot

MiquelRForgeFlow committed:
[FIX] stock_picking_type: assure api.multi in write

MiquelRForgeFlow committed:
[FIX] quality_control_stock: singleton error

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-11.0/manufacture-11.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-11-0/manufacture-11-0-mrp_production_putaway_strategy/de/

JordiBForgeFlow committed:
[quality_control_stock][FIX] the stock user should be able to
create quality control inspection without necessarily being
quality control user. It is common that the person doing the
picking is not going to be the same as the one doing the inspection.

So when a stock user create the transfer, the system will create
the necessary qc inspections, regardless if the user has permissions
to process them.

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (27 of 27 strings)

Translation: manufacture-11.0/manufacture-11.0-quality_control_stock
Translate-URL: https://translation.odoo-community.org/projects/manufacture-11-0/manufacture-11-0-quality_control_stock/de/

mvrodriguez committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (27 of 27 strings)

Translation: manufacture-11.0/manufacture-11.0-quality_control_stock
Translate-URL: https://translation.odoo-community.org/projects/manufacture-11-0/manufacture-11-0-quality_control_stock/es/

HviorForgeFlow committed:
[MIG] mrp_production_putaway_strategy: Migration to 12.0

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update mrp_production_putaway_strategy.pot

NachoAlesLopez committed:
[MIG] quality_control_stock: Migration to 12.0

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-12.0/manufacture-12.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-mrp_production_putaway_strategy/

OCA-git-bot committed:
[UPD] README.rst

kos94ok-3D committed:
[IMP] quality_control_stock: use read_group

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-12.0/manufacture-12.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-mrp_production_putaway_strategy/pt_BR/

sudhir-erpharbor committed:
[MIG] Migrated mrp_production_putaway_strategy to v13.

LoisRForgeFlow committed:
[IMP] mrp_production_putaway_strategy: black, isort, prettier

ps-tubtim committed:
[IMP] quality_control_stock: black, isort

ps-tubtim committed:
[MIG] quality_control_stock: Migration to 13.0

oca-travis committed:
[UPD] Update mrp_production_putaway_strategy.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-13.0/manufacture-13.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-13-0/manufacture-13-0-mrp_production_putaway_strategy/

ortlam committed:
Translated using Weblate (German)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-13.0/manufacture-13.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-13-0/manufacture-13-0-mrp_production_putaway_strategy/de/

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-13.0/manufacture-13.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-13-0/manufacture-13-0-mrp_production_putaway_strategy/

MiquelRForgeFlow committed:
[UPD] Eficent -> ForgeFlow

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[UPD] README.rst

CarlosRoca13 committed:
[FIX] quality_control_stock_oca: Fixed inheritance and module renamed

[UPD] Update quality_control_stock_oca.pot

[UPD] README.rst

pedrobaeza committed:
[FIX] quality_control_stock_oca: Avoid singleton error

A compute must be multi-record aware

enriquemartin committed:
[MIG] quality_control_stock_oca:  Migration to 14.0

jonerikceberio committed:
[IMP] quality_control_stock_oca: sudo the process of creating the test and calculated fields, so no permissions have to be assigned to the sotck user

[UPD] Update quality_control_stock_oca.pot

[UPD] README.rst

[IMP] update dotfiles [ci skip]

JoanMForgeFlow committed:
[14.0][MIG] mrp_production_putaway_strategy: Migration to 14.0

JoanMForgeFlow committed:
[15.0][MIG] mrp_production_putaway_strategy: Migration to 15.0

Commit:
[UPD] Update mrp_production_putaway_strategy.pot

OCA-git-bot committed:
[UPD] README.rst

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-15.0/manufacture-15.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_production_putaway_strategy/

badbole committed:
fixed singleton error

quality_control_stock_oca 14.0.1.0.1

ThiagoMForgeFlow committed:
[14.0][ADD] mrp_unbuild_subcontracting: unbuild created when is returned a product subcontracted

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: manufacture-14.0/manufacture-14.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-quality_control_stock_oca/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_production_putaway_strategy/it/

Commit:
[UPD] Update mrp_unbuild_subcontracting.pot

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (13 of 13 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_unbuild_subcontracting
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_unbuild_subcontracting/it/

dalonsod committed:
[MIG] quality_control_stock_oca: Migration to 15.0

JordiBForgeFlow committed:
[FIX] mrp_unbuild_subcontracting: raise error when the subcontracting move
is linked with more than one manufacturing order.

ChrisOForgeFlow committed:
fix: partial receipt and refund of subcontracting purchases

angelmoya committed:
[MIG] quality_control_stock_oca: Migration to 16.0

angelmoya committed:
[MIG] quality_control_stock_oca: Migration to 16.0

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_production_putaway_strategy/es/

OCA-git-bot committed:
[UPD] README.rst

PauBForgeFlow committed:
[MIG] mrp_unbuild_subcontracting: Migration to 16.0

dalonsod committed:
[IMP] quality_control_stock_oca: picking types - use display_name for trigger name

[UPD] Update quality_control_stock_oca.pot

[BOT] post-merge updates

ChrisOForgeFlow committed:
[14.0][FIX] added case for partial receipt on purchase of subcontrating

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-15.0/manufacture-15.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_production_putaway_strategy/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_production_putaway_strategy
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_production_putaway_strategy/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-quality_control_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: manufacture-15.0/manufacture-15.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-quality_control_stock_oca/it/

Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: manufacture-15.0/manufacture-15.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-quality_control_stock_oca/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (162 of 162 strings)

Translation: manufacture-16.0/manufacture-16.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-quality_control_oca/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (162 of 162 strings)

Translation: manufacture-16.0/manufacture-16.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-quality_control_oca/it/

pedrobaeza committed:
[FIX] quality_control_oca: XML-ID must be fully qualified

Fixes #1167

OCA-git-bot committed:
[BOT] post-merge updates

BernatPForgeFlow committed:
[FIX] mrp_unbuild_subcontracting: Adapt logic for partial receipts in 16.0

victoralmau committed:
[MIG] quality_control_stock_oca: Migration to 16.0

TT47858

victoralmau committed:
[MIG] quality_control_stock_oca: Migration to 16.0

TT47858

Commit:
[UPD] Update quality_control_stock_oca.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-quality_control_stock_oca/

quentinDupont committed:
[ADD] mrp_bom_select_product_variant

uolaizola committed:
[FIX] quality_control_oca add colspan attribute to show the inline tree view of the fields correctly

OCA-git-bot committed:
[BOT] post-merge updates

ChrisOForgeFlow committed:
[FIX] mrp_unbuild_subcontracting, action_validate expect to be singleton

ChrisOForgeFlow committed:
Update mrp_unbuild_subcontracting/models/stock_picking.py

You are right, thank you

Co-authored-by: Yoshi Tashiro (Quartile) <tashiro@quartile.co>

Commit:
[IMP] quality_control_oca: pre-commit auto fixes

Commit:
[MIG] quality_control_oca: Migration to 17.0

Commit:
[IMP] quality_control_stock_oca: pre-commit auto fixes

Commit:
[MIG] quality_control_stock_oca: Migration to 17.0

Commit:
[UPD] Update quality_control_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

BernatPForgeFlow committed:
[FIX] mrp_unbuild_subcontracting: Cannot unbuild subcontracting orders with many links

From version 16.0, when we create backorders we are not splitting the Origin Moves. For that reason, when we try to unbuild a subcontracting order that has more than origin move we cannot know which is the real origin.

There has been a PR to fix this in Odoo here: https://github.com/odoo/odoo/pull/148262, but it was reverted after some time due to some problems on the splitting. If we want to allow this, we would need to propose a PR in Odoo that completely fixes this problem.

jakobkrabbe committed:
Translated using Weblate (Swedish)

Currently translated at 100.0% (162 of 162 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_oca/sv/

AungKoKoLin1997 committed:
[IMP] quality_control_oca, quality_control_stock_oca

This commit adds date_done field, update views and filters and the timing field in the QC trigger line to enable
following scenarios:

- When timing is 'Before', an inspection is generated for each related
  move when a picking with the trigger is confirmed.
- When timing is 'Plan Ahead', a 'Plan' inspection is generated for
  each related move when a picking with the trigger is confirmed. A
  plan inspection is just a plan, and cannot be updated except for the
  date. A plan inspection gets converted into an executable inspection
  once the picking is done.

Commit:
[UPD] Update quality_control_stock_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/

Commit:
[UPD] Update quality_control_oca.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-17.0/manufacture-17.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/it/

jans23 committed:
Translated using Weblate (German)

Currently translated at 74.3% (119 of 160 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_oca/de/

Commit:
[UPD] Update quality_control_oca.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-17.0/manufacture-17.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (24 of 24 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/it/

Trivedi-Vacha-SerpentCS committed:
[Mig][18.0] quality_control_oca: Migrated the module

Commit:
[UPD] Update mrp_unbuild_subcontracting.pot

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-mrp_unbuild_subcontracting
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_unbuild_subcontracting/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_unbuild_subcontracting
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_unbuild_subcontracting/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_bom_select_product_variant
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_bom_select_product_variant/it/

BhaveshHeliconia committed:
[IMP] mrp_bom_select_product_variant: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] mrp_bom_select_product_variant: Migration to 18.0

LauraCForgeFlow committed:
[FIX] quality_control_oca: notes width in QC inspection form

manuelregidor committed:
[IMP]quality_control_stock_oca: inspection per lot/serial

AungKoKoLin1997 committed:
[ADD] mrp_production_allow_recursive

xaviedoanhduy committed:
[IMP] mrp_unbuild_subcontracting: pre-commit auto fixes

xaviedoanhduy committed:
[MIG] mrp_unbuild_subcontracting: Migration to 18.0

nguyenminhchien commented on mrp_unbuild_subcontracting/models/stock_picking.py:
```diff
@@ -0,0 +1,112 @@
+                ).action_validate()
+            moves = picking.move_ids.filtered("is_subcontract")
+            finished_move = unbuilds_to_done.produce_line_ids.filtered(
+                lambda m: m.product_id.id in moves.mapped("product_id").ids  # noqa: B023
```
```suggestion
                lambda m, ms=moves: m.product_id.id in ms.mapped("product_id").ids
```

diggy128 committed:
[16.0][IMP] quality_control_oca: Add kanban views for qc.inspection and qc.inspection.line

diggy128 committed:
[IMP] quality_control_oca: update methods signatures

Update create signature to use vals_list
Fix raise in unlink override. Replace with api.ondelete method
Allow superuser to override ondelete constraint for autogenerated inspection
Replace unnecessary elif

Commit:
[UPD] Update quality_control_stock_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (26 of 26 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/it/

AaronHForgeFlow committed:
[MIG] quality_control_oca: Migration to v18

Commit:
[UPD] Update mrp_production_allow_recursive.pot

OCA-git-bot committed:
[BOT] post-merge updates

LauraCForgeFlow committed:
[FIX] quality_control_stock_oca: qc_inspections_ids access from picking

AungKoKoLin1997 committed:
[IMP] quality_control_oca, quality_control_stock_oca
This commit adds date_done field, update views and filters and the timing field in the QC trigger line to enable
following scenarios:

- When timing is 'Before', an inspection is generated for each related
  move when a picking with the trigger is confirmed.
- When timing is 'Plan Ahead', a 'Plan' inspection is generated for
  each related move when a picking with the trigger is confirmed. A
  plan inspection is just a plan, and cannot be updated except for the
  date. A plan inspection gets converted into an executable inspection
  once the picking is done.

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (6 of 6 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_production_allow_recursive
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_production_allow_recursive/it/

LauraCForgeFlow committed:
[FIX] quality_control_stock_oca: multiple qc_trigger lines

Kev-Roche committed:
[16.0][IMP][quality_control_oca] avoid test question without value

Commit:
[UPD] Update quality_control_stock_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/

remi-filament committed:
Translated using Weblate (French)

Currently translated at 100.0% (27 of 27 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/fr/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (27 of 27 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/it/

remi-filament committed:
Translated using Weblate (French)

Currently translated at 100.0% (27 of 27 strings)

Translation: manufacture-17.0/manufacture-17.0-quality_control_stock_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-quality_control_stock_oca/fr/

ArnauCForgeFlow committed:
[IMP] mrp_production_putaway_strategy: pre-commit auto fixes

ArnauCForgeFlow committed:
[MIG] mrp_production_putaway_strategy: Migration to 18.0

OCA-git-bot committed:
Merge PR #1575 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_production_picking_type_from_route.pot

OCA-git-bot committed:
[BOT] post-merge updates

JoanSForgeFlow committed:
[ADD] mrp_production_location_picking_type

OCA-git-bot committed:
Merge PR #1574 into 18.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update mrp_attachment_mgmt.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1576 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_production_putaway_strategy.pot

OCA-git-bot committed:
[BOT] post-merge updates

Commit:
[UPD] Update mrp_production_location_picking_type.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1523 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update quality_control_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

eugenios73 committed:
[18.0][FIX] mrp_bom_attribute_match

OCA-git-bot committed:
Merge PR #1583 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

bo-eska committed:
Added translation using Weblate (Turkish)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (161 of 161 strings)

Translation: manufacture-18.0/manufacture-18.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-quality_control_oca/it/

florentx committed:
[ADD] printing_auto_mrp

OCA-git-bot committed:
Merge PR #1551 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_bom_hierarchy.pot

OCA-git-bot committed:
[BOT] post-merge updates

florentx committed:
[IMP] printing_auto_mrp: more tests

bo-eska committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (17 of 17 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_bom_attribute_match
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_bom_attribute_match/tr/

corredato committed:
[FIX] mrp_bom_hierarchy: bom lines read permission

OCA-git-bot committed:
Merge PR #1585 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update printing_auto_mrp.pot

OCA-git-bot committed:
[BOT] post-merge updates

LoisRForgeFlow committed:
[IMP] mrp_multi_level: add RfQ quantity and button to check RfQs

It is useful to know that the supply quantity might not be confirmed yet
directly from the MRP inventory screen.

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-18.0/manufacture-18.0-printing_auto_mrp
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-printing_auto_mrp/it/

grindtildeath committed:
[IMP] mrp_lot_number_propagation: Support mass production

bo-eska committed:
Added translation using Weblate (Turkish)

bo-eska committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_sale_info/tr/

OCA-git-bot committed:
Merge PR #1595 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_multi_level.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-18.0/manufacture-18.0-mrp_multi_level
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_multi_level/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (172 of 172 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_multi_level
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_multi_level/it/

AaronHForgeFlow committed:
[MIG] quality_control_stock_oca: Migration to v18

OCA-git-bot committed:
Merge PR #1596 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update quality_control_stock_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1599 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1592 into 18.0

Signed-off-by grindtildeath

Commit:
[UPD] Update mrp_lot_number_propagation.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-18.0/manufacture-18.0-mrp_lot_number_propagation
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_lot_number_propagation/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (42 of 42 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_lot_number_propagation
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_lot_number_propagation/it/

AungKoKoLin1997 committed:
[IMP] mrp_production_allow_recursive: pre-commit auto fixes

AungKoKoLin1997 committed:
[MIG] mrp_production_allow_recursive: Migration to 18.0

bo-eska committed:
Added translation using Weblate (Turkish)

bo-eska committed:
Translated using Weblate (Turkish)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_bom_location
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_bom_location/tr/

OCA-git-bot committed:
Merge PR #1605 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update quality_control_oca.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-18.0/manufacture-18.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-quality_control_oca/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (169 of 169 strings)

Translation: manufacture-18.0/manufacture-18.0-quality_control_oca
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-quality_control_oca/it/

Saran440 committed:
[FIX] quaility_control_oca: invisible column is not used

LoisRForgeFlow committed:
[IMP] mrp_multi_level: add url path to main actions

OCA-git-bot committed:
Merge PR #1609 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

AaronHForgeFlow committed:
[FIX] quality_control_stock_oca: super() call to _create_backorder()

Vang-NguyenPhu committed:
[18.0][FIX] mrp_bom_attribute_match: Error when use Compute Price from BOM on Product use Attribute Match on BOM Lines

OCA-git-bot committed:
Merge PR #1612 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1608 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

bealdav committed:
FIC mrp_bom_section_and_note: consu instead of product in ctx

uvid-vlado committed:
Added translation using Weblate (Croatian (hr_HR))

uvid-vlado committed:
Added translation using Weblate (Croatian)

uvid-vlado committed:
Translated using Weblate (Croatian)

Currently translated at 80.0% (8 of 10 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_sale_info/hr/

eugenios73 committed:
Added translation using Weblate (Spanish)

eugenios73 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (4 of 4 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_production_back_to_draft
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_production_back_to_draft/es/

OCA-git-bot committed:
Merge PR #1616 into 18.0

Signed-off-by quentinDupont

OCA-git-bot committed:
[BOT] post-merge updates

JasminSForgeFlow committed:
[IMP] mrp_production_location_picking_type: black, isort, prettier

JasminSForgeFlow committed:
[MIG] mrp_production_location_picking_type: Migration to 18.0

LauraCForgeFlow committed:
[FIX] mrp_multi_level: MRP areas in stock location groups

OCA-git-bot committed:
Merge PR #1628 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

AaronHForgeFlow committed:
[FIX] mrp_multi_level: user_has_groups call is deprecated

OCA-git-bot committed:
Merge PR #1635 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1624 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_production_location_picking_type.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

OCA-git-bot committed:
Merge PR #1435 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_bom_select_product_variant.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (4 of 4 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_production_location_picking_type
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_production_location_picking_type/it/

OCA-git-bot committed:
Merge PR #1602 into 18.0

Signed-off-by ivantodorovich

Commit:
[UPD] Update mrp_production_allow_recursive.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1614 into 18.0

Signed-off-by ivantodorovich

Commit:
[UPD] Update mrp_bom_attribute_match.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-18.0/manufacture-18.0-mrp_bom_attribute_match
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_bom_attribute_match/

OCA-git-bot committed:
Merge PR #1549 into 18.0

Signed-off-by ivantodorovich

Commit:
[UPD] Update mrp_restrict_lot.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (18 of 18 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_bom_attribute_match
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_bom_attribute_match/it/
